src/components/atoms/seo.js

```diff
@@ -37,20 +37,16 @@ function SEO({ meta }) {
     `
   )
 
-  const description = meta.description
-    ? meta.description
-    : site.siteMetadata.description
+  const description = site.siteMetadata.description
```

Comment:
sorry, might have been some confusion on my request here.  we still want to keep the option for page level meta descriptions, but if there is not one specified in the page frontmatter it should use the site default.  i think this is how it was configured before this change, and we'd just need to remove the descriptions from the frontmatter to make this happen?

Reply:
@Lane should I leave that block as is? with:
`
const description = meta.description    ? meta.description    : site.siteMetadata.description`

also, should I take the description field out of the frontmatter entirely? just wanna make sure i understand what you're suggesting

Reply:
yep, leave as is and then remove generic `Page description` values from frontmatter (but leave the key)

e.g. 

```
description: Page description
```

becomes:
```
description:
```

without a value specified, it should use site default, but if they enter a value it should use that.